### ISSUE TITLE
--no-colors is being appended to Script Parameters

### ISSUE THREAD

**Issue by njlbenn** [pantsbuild/intellij-pants-plugin#112]
Hi

I am setting up a Run Configuration to run one of my applications inside IntelliJ for debugging.

I need to pass some flags to my application so I am using the "Script parameters" field in the Run Configuration.

So let's say I have an application called `hello` that I want to pass a `name` flag, then I am hoping for something like:

```
pants run src/scala/hello:app -- -name=Joe
```

So in the "Scripts parameters" filed I have pasted `-- -name=Joe`

The problem seems to be that when I click to run it in IntelliJ the task runner appends a `--no-colors` flag at the *end* of my script parameters. So the application effectively gets called like:

```
pants run src/scala/hello:app -- -name=Joe --no-colors
```

This causes some "unrecognised flag" issues in the application. Of course I can hack around this in the application, but I just thought I would raise this as a small bug that maybe is easy for you to fix and save me and others a minor inconvenience :).

I guess final output from the task runner should be something like the following, where the --no-colors comes before the script parameters?

```
pants run src/scala/hello:app --no-colors -- -name=Joe
```

Alternatively, maybe I am not using the Run Configuration dialog correctly?

**Comment by maxim111333** [comment]
I am not appending any custom parameters but still getting this error when trying to run tests:

`Exception message: Unrecognized command line flags on global scope: --no-color`

Any hints on how to workaround this?

**Comment by wisechengyi** [comment]
@maxim111333 Given --no-color was appended instead of --no-colors, likely you need to upgrade the plugin, it was fixed months ago. 

regarding the script parameters, it is a bug and thanks for bringing this up. https://github.com/wisechengyi/intellij-pants-plugin/blob/master/src/com/twitter/intellij/pants/service/task/PantsTaskManager.java#L67-L78
script parameters are appended before --no-colors. 

I can't think of a workaround in pants run configuration. You may try using the intellij junit runner (it only uses pants for compile), so it should take your parameters correctly if it takes any. If there is already a pants run configuration created for a particular test, it needs to be deleted before both choices will show up again.

![example]
(https://github.com/pantsbuild/intellij-pants-plugin/blob/master/images/tasks/create_task_from_context_all.png)

**Comment by wisechengyi** [comment]
https://rbcommons.com/s/twitter/r/3544

**Comment by wisechengyi** [comment]
fixed at 1.3.14 https://plugins.jetbrains.com/plugin/7412
